api/k8s/v1/k8s.proto

```diff
@@ -319,6 +335,56 @@ message DeleteDeploymentRequest {
 message DeleteDeploymentResponse {
 }
 
+message Service {
+  option (clutch.api.v1.id).patterns = {
+    type_url : "clutch.k8s.v1.Service",
+    pattern : "{cluster}/{namespace}/{name}"
+  };
+
+  string cluster = 1;
+  string namespace = 2;
+  string name = 3;
+  string type = 4;
```

Comment:
should type be an enum?

Reply:
Yeah here are the types available 
https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types

Reply:
Done. Addressed in this commit https://github.com/lyft/clutch/pull/640/commits/e4fd85020d2ccccdddab4d0749e7e899b48ee9de